support/Pipelines/EccentricityControl/EccentricityControl.py

```diff
@@ -120,10 +120,7 @@ def compute_separation(h5_file, subfile_name_aha, subfile_name_ahb):
         )
 
     # Compute separation
-    num_obs = len(ObjectA_centers[:, 0])
-
-    if len(ObjectB_centers[:, 0]) < num_obs:
-        num_obs = len(ObjectB_centers[:, 0])
+    num_obs = min(len(ObjectA_centers[:, 0]), len(ObjectB_centers[:, 0]))
```

Comment:
Maybe this commit and the one before should be in a separate PR since they are a bug fix? That way they don't get buried in an experimental PR